packages/eas-cli/src/commandUtils/context/contextUtils/getProjectIdAsync.ts

```diff
@@ -191,15 +191,15 @@ export async function validateOrSetProjectIdAsync({
         return user.username;
       case 'Robot':
         throw new Error(
-          'The "owner" manifest property is required when using robot users. See: https://docs.expo.dev/versions/latest/config/app/#owner'
+          "Must configure EAS project by running 'eas init' before using a robot user to manage the project."
```

Comment:
Put the command in double quotes:
```suggestion
          'Must configure EAS project by running "eas init" before using a robot user to manage the project.'
```